sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/EmployeeService.java

```diff
@@ -0,0 +1,18 @@
+package com.exadel.sandbox.team5.service;
+
+import com.exadel.sandbox.team5.entity.Employee;
+
+import java.util.List;
+
+public interface EmployeeService {
```

Comment:
Please extract the common `CRUDService<T extend BaseEntity>` or `EntityCRUDService<T extend BaseEntity>` interface with generic type which extends from `BaseEntity`

Reply:
I must refactor this interface or create new?

Reply:
Let's create a new one. And leave it as `EmployeeService extends CRUDService<Employee>`.